plugins/authentication/src/OAuthModel/model.tsx

```diff
@@ -275,8 +275,7 @@ const stateModelFactory = (configSchema: OAuthInternetAccountConfigModel) => {
 
           const eventName = `JBrowseAuthWindow-${self.internetAccountId}`
           if (isElectron) {
-            const electron = require('electron')
-            const { ipcRenderer } = electron
+            const { ipcRenderer } = window.require('electron')
```

Comment:
this was a recommendation from a thread (forget where) on how to import electron ipcRenderer, used elsewhere in this PR too but avoids attempts at being bundled

Reply:
if we don't do this get this error

`Uncaught TypeError: fs.existsSync is not a function` from require/import electron

Reply:
unfortunately i haven't gotten jbrowse-desktop test suite in products/jbrowse-desktop to work (window.require is undefined in jest) with this yet because the require/import was needed for the electron mock that was in place, so products/jbrowse-desktop/src/JBrowse.test.tsx deleted for now

Reply:
restored test :)

Reply:
mocks window.require in the packages/__mocks__/electron